### PR TITLE
ref(*): change Name to Installation on Claim

### PR DESCRIPTION
**Pull request by vdice**
* change `Name` to `Installation` on a `Claim`

Ref https://github.com/cnabio/cnab-spec/pull/310

### CODE REVIEW

File: claim/claim.go [hunk 1]
```diff
@@ -35,13 +35,13 @@ const (
...
 // a CNAB package.
 type Claim struct {
-	Name       string                 `json:"name"`
-	Revision   string                 `json:"revision"`
-	Created    time.Time              `json:"created"`
-	Modified   time.Time              `json:"modified"`
-	Bundle     *bundle.Bundle         `json:"bundle"`
-	Result     Result                 `json:"result,omitempty"`
-	Parameters map[string]interface{} `json:"parameters,omitempty"`
+	Installation string                 `json:"installation"`
```
Comment (carolynvs) [inline claim/claim.go]: We put the spec version in bundle.json so tools can tell what version of the spec it has been serialized to match. 

Seeing a breaking change like this and thinking through how tools can detect and migrate existing claims gracefully while also supporting the new claim format, I think we should also start putting the schema version for the claims spec on the `Claim` struct as well.

That way a tool can deserialize to a map or scan the json document for schema version, figure out if it's the old one with `name` or the new one with `installation`, handle the migration if needed, then read it into the new cnab-go struct.

I'll make a new issue in cnab-go for that.

File: claim/claim_test.go [hunk 2]
```diff
@@ -132,7 +132,7 @@ func TestClaimSchema(t *testing.T) {
 	claimBytes, err := json.Marshal(exampleClaim)
 	assert.NoError(t, err, "failed to json.Marshal the claim")
 
-	url := "https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/claim.schema.json"
+	url := "https://cnab.io/v1/claim.schema.json"
```
Comment (carolynvs) [inline claim/claim_test.go]: 🎉

File: claim/claim.go (hunk 1 above)
Comment (carolynvs) [reply claim/claim.go]: Ok, I created #182

**Comment by vdice** [comment]
@carolynvs are you good with this PR as it is?  Alas, as mentioned, this breaking change doesn't really prompt a claim version change from `v1.0.0-WD` since it's still in the working draft phase...

**Comment by carolynvs** [comment]
Yup! Sorry for the confusion, that's why I created the follow-up issue. 👍

**Pull request merged**